src/polyswarmd/rpc.py

```diff
@@ -1,13 +1,94 @@
 import time
-import gevent
-import json
+from typing import Optional
 
-from gevent.lock import BoundedSemaphore
+import web3.eth
+from web3.utils import Filter
 from requests.exceptions import ConnectionError
-from polyswarmartifact.schema import Bounty as BountyMetadata
-from requests_futures.sessions import FuturesSession
 
-from polyswarmd.utils import *
+import gevent
+from gevent.lock import BoundedSemaphore
+from gevent.pool import Pool
+from polyswarmd.utils import logging
+from polyswarmd.websockets.messages import (WebsocketEventlogMessage, Deprecated, FeesUpdated, InitializedChannel,
```

Comment:
I am not seeing Deprecated, NewBounty, QuorumReached, SettledBounty, or RevealAssertion in websockets.messages.